docs/api.md

```diff
@@ -62,6 +62,8 @@ interface NeuralNetworkContext {
   const long MAXIMUM = 65;
   const long ATROUS_CONV_2D = 10003;
   const long ATROUS_DEPTHWISE_CONV_2D = 10004;
+  const long ARGMAX = 10005;
+  const long ARGMIN = 10006;
```

Comment:
Could you please align the constant value to NNAPI? 
```
const long ARGMAX = 39;
const long ARGMIN = 40;
```

Reply:
Thanks, updated them by the new patch.